README.md

```diff
@@ -1,16 +1,58 @@
 # Exercises
 
-__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM! 
+__NOTE:__ DO NOT USE THE BUILT IN METHODS WHEN ASKED TO MAKE THEM!
 
 1. Using Array.prototype add a new method `myForEach` that mimics the behaivor of the built in `forEach`.
 
-2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`. 
+Array.prototype.myForEach = function (callback) {
+  for (let i = 0; i < this.length; i++) {
+    callback(this[i]);
+  }
+};
 
-3. Write a 'myFilter` that behave's the same as `filter`. 
+2. Use your `myForEach` to add `myMap` to the Array prototype. `myMap` should behave the same as regular `map`.
 
-4. Write a `myEvery`. 
+Array.prototype.myMap = function (callback) {
+  let result = [];
+  for (this i = 0; i < this.length; i++) {
+    result += this[i]
+  }
+};
+callback(result);
 
-5. Write a `myReduce`. It should use the first element in the array if none is given. 
+3. Write a 'myFilter' that behave's the same as `filter`.
+
+let result = [];
+Array.prototype.myFilter = function(callback) {
+  for (let i = 0; i < this.length; i++) {
+    if (this[i]) {
+      result.push(this[i])
+    }
+  }
+};
+callback(result);
+
+4. Write a `myEvery`.
+
+let result = true
+Array.prototype.myEvery = function (callback) {
+  for (let i = 0; i < this.length; i++) {
+    if(this[i]) {
+      result = false
+    }
+  }
+};
+callback(result)
+
+5. Write a `myReduce`. It should use the first element in the array if none is given.
+
+Array.prototype.myReduce = function (callback) {
+  let result = [];
+  for (let i = 0; i < this.length; i++) {
+    result += this[i];
```

Comment:
How would you implement this with a second argument representing the accumulator?